File: Android/aryanmishra29_AryanMishra_2125csme1029_2/GettingStarted/Reverse_A_Number.java
import java.util.Scanner;

public class Reverse_A_Number {
   public static void main(String[] args) {
     // write your code here 
     Scanner sc = new Scanner(System.in);
     int n=sc.nextInt();
     while(n>0){
         System.out.println(n%10);
         n=n/10;
     }
    }
   
}

File: Web/kashish12_kashishalikhan_2125cse1133_02/GettingStarted/reverseanumber.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    int p;
    while(n != 0){
        p = n % 10;
        cout<<p<<"\n";
        n = n / 10;
    }

    //write your code here
    return 0;
}